test/unit/testSession.test.ts

```diff
@@ -180,7 +204,8 @@ describe('TestSession', () => {
       expect(session.dir).to.equal(path.join(cwd, `test_session_${session.id}`));
       expect(session.homeDir).to.equal(session.dir);
       expect(session.project).to.equal(undefined);
-      expect(session.setup).to.deep.equal([execRv]);
+      (session.setup ?? []).forEach((setup) => (setup.execCmdDuration = Duration.milliseconds(0)));
```

Comment:
Could not find a simple solution to stubbing global function, so forcing duration to 0.